lib/grpc.js

```diff
@@ -170,7 +171,13 @@ GrpcClient.prototype.createStub = function(
   }).then(function(auth) {
     return self._getCredentials(options, auth);
   }).then(function buildStub(credentials) {
-    return new CreateStub(serviceAddress, credentials);
+    var grpcOptions = {};
+    Object.keys(options).forEach(function(key) {
+      if (key.indexOf('grpc.') == 0) {
```

Comment:
It would be good to add a test to make sure that the correct keys are being extracted from options.